addons/web/static/src/core/l10n/localization_service.js

```diff
@@ -54,7 +54,10 @@ export const localizationService = {
             }
         }
 
-        Object.setPrototypeOf(translatedTerms, terms);
+        // Object.setPrototypeOf(translatedTerms, terms);
+        Object.assign(translatedTerms, terms);
+        translatedTerms.__isLoaded__ = true;
```

Comment:
The reason I suggested to write this on _t instead of on the translated terms is that this creates a translation entry for the string `__isLoaded__` where the translation is `true`. If you want to put it on the terms, you should use a `Symbol` to avoid possible conflicts.

Reply:
Agree ! I'll use Symbol